test/integration/test_integration_basics.py

```diff
@@ -247,47 +247,58 @@ def test_https_h2(https_test_server_fixture):
   assertEqual(len(counters), 17)
 
 
+def _do_tls_configuration_test(https_test_server_fixture, cli_parameter, use_h2):
```

Comment:
Can we document the arguments as per our Python style guide?

```python
def _do_tls_configuration_test(https_test_server_fixture, cli_parameter, use_h2):
  """Runs tests for different ciphers.

  For a given choice of (--tls-context, --transport-socket) x (H1, H2),
  run a series of traffic tests with different ciphers.

  Args:
    https_test_server_fixture: type, what it is ...
    ...
  """
```

Reply:
Done